lxd/config/map.go

```diff
@@ -73,8 +71,10 @@ func (m *Map) Change(changes map[string]string) (map[string]string, error) {
 func (m *Map) Dump() map[string]string {
 	values := map[string]string{}
 
+	m.schema.RLock()
+	defer m.schema.RUnlock()
```

Comment:
It's probably unlikely that the `Dump` function will grow significantly but I'm wondering if the unlock call should not be `defer`'ed and simply put right after the `for` loop?

Maybe I'm overthinking it in which case I'd blame https://go.dev/blog/flight-recorder#example ;)